slo-monitor/vendor/github.com/imdario/mergo/README.md

```diff
@@ -79,7 +79,6 @@ If Mergo is useful to you, consider buying me a coffee, a beer or making a month
 - [elwinar/rambler](https://github.com/elwinar/rambler)
 - [tmaiaroto/gopartman](https://github.com/tmaiaroto/gopartman)
 - [jfbus/impressionist](https://github.com/jfbus/impressionist)
-- [Jmeyering/zealot](https://github.com/Jmeyering/zealot)
```

Comment:
This is vendor directory managed by go modules. This particular dependency is coming from github.com/imdario/mergo library. It should be fixed there and then that dependency should be updated.